protocol-units/settlement/mcr/client/src/eth_client.rs

```diff
@@ -57,6 +57,23 @@ sol!(
 	"abis/MCRLegacy.json"
 );
 
+// When created, kill the pid when dropped.
+// Use to kill Anvil process when Suzuka Node end.
+// TODO should be removed by new config.
+struct AnvilKillAtDrop {
+	pid: u32,
+}
+
+impl Drop for AnvilKillAtDrop {
+	fn drop(&mut self) {
+		tracing::info!("Killing Anvil process");
+		if let Err(err) = std::process::Command::new("kill").args(&[&self.pid.to_string()]).spawn()
+		{
+			tracing::info!("warn, an error occurs during Anvil process kill : {err}");
+		}
+	}
+}
```

Comment:
1. This a very happy-path solution for minding a child process. Let's leave it to process-compose or whatever other systemic solution we come up with.
2. There are in-process ways to kill a process, no need to run a command.
3. This is also not portable to non-Unix systems.

Reply:
I'm agree. I do it like that because, in the std lib I didn't find a way to do it and I didn't want to import a new crate because it's a temporary solution until we settlement the config and process management. So it's working for mac and linux that are the current OS that we target that why I use this very easy solution.